docs/preview/gha-runner-scale-set-controller/README.md

```diff
@@ -102,13 +102,12 @@ https://user-images.githubusercontent.com/568794/212668313-8946ddc5-60c1-461f-a7
     name: Test workflow
     on:
         workflow_dispatch:
-
     jobs:
-    test:
-        runs-on: arc-runner-set
-        steps:
-        - name: Hello world
-            run: echo "Hello world"
+        test:
```

Comment:
I thought YAML used 2 instead of 4-spacce indentation!

Reply:
This is a 4-spacce indentation.
Where am I wrong?

Reply:
@kahirokunn you have 2 additional spaces 😄 it's a nit but I agree with @mumoshu - all our workflows use 2 space indentation for yaml. Even if the original was wrong, let's correct it since we're already at it.

Reply:
okay. done. 👍